### DISCUSSION TITLE
Change the ia_pd example in dhcpcd.conf(5) to use the ID 0 for ia_na and the first ia_pd

### DISCUSSION THREAD

**Discussion by zacknewman** [rsmarples/dhcpcd#209] [General]
There are a lot of misconceptions about DHCPv6 and DHCP that you can find all over the web (e.g., servers and clients using a specific UDP source port). There are two misconceptions about DHCPv6 that I see frequently: IAIDs cannot be 0 and IA_NA and IA_PD IDs must be different from each other. RFC 8415 explicitly uses the ID 0 as a valid value. Additionally it states "The IAID uniquely identifies the IA and MUST be chosen to be unique among the IAIDs for that IA type on the client (e.g., an IA_NA with an IAID of 0 and an IA_PD with an IAID of 0 are each considered unique)." I feel like `dhcpcd.conf(5)` inadvertently promotes these misconceptions by not using the ID 0 and using distinct values for IA_NA and the first IA_PD. While the man page is not incorrect, I would like to dispel these notions by changing the example shown for `ia_pd` to be:
```bash
    noipv6rs                 # disable routing solicitation
    denyinterfaces eth2      # Don't touch eth2 at all
    interface eth0
        ipv6rs                 # enable routing solicitation for eth0
        ia_na 0                # request an IPv6 address
        ia_pd 0 eth1/0         # request a PD and assign it to eth1
        ia_pd 1 eth2/1 eth3/2  # req a PD and assign it to eth2 and eth3
```

**Comment by Kentzo** [comment]
Another thing that worths emphasizing is why "IPv6RS should be disabled globally when requesting a Prefix Delegation". It's not clear why it should be off for non-delegated interfaces and why it cannot be on for delegated interfaces to receive other configuration.

**Comment by ido** [comment]
If you'd like to submit a PR to clarify the man page, feel free to do so. Resolving for now unless there is anything further on this topic to discuss.
